Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -167,12 +161,100 @@ private fun CartContent(
             )
         }
         items(orderLines) { orderLine ->
-            CartItem(
-                orderLine = orderLine,
-                removeSnack = removeSnack,
-                increaseItemCount = increaseItemCount,
-                decreaseItemCount = decreaseItemCount,
-                onSnackClick = onSnackClick
+            SwipeDismissItem(
+                background = { offsetX ->
+                    val paddingRight = when {
+                        offsetX == 0f || offsetX < -300f -> 0.dp
```

Comment:
Please can you add some comments for each branch

Reply:
I have already added the requested comments